config/routes.yaml

```diff
@@ -12,8 +12,4 @@ sitemap:
 
 controllers:
     resource: ../src/Controller/
-    type: annotation
-
-kernel:
-    resource: ../src/Kernel.php
-    type: annotation
+    type: attribute
```

Comment:
Did you change the controllers too? :thinking:

Reply:
We already use attributes :)